apache/couchdb-fauxton
# Customize style of the results list

## Overview

Adds a new menu to customize the style of the results list.

 - Truncated/Show full values
 - Font size: small, medium, large

![image](https://user-images.githubusercontent.com/30349380/48089115-b374f080-e1d1-11e8-86b6-7f54477cde80.png)


## Testing recommendations

 - Go to documents list and switch to the Metadata view
 - Click the new button to show the menu
 - Click on `Show full values`
   - The full text value is displayed for each row
   - Menu label changes to 'Truncate values'
 - Click on `Small`, `Medium`, `Large` and verify the font size is updated
 - Switch to the Table view, and verify the same options also work
 - Switch to the JSON view, the menu should only show the 'Font size' options

## Checklist
